README.md

```diff
@@ -103,10 +118,28 @@ Log into Docker as above if necessary, then run:
   docker push nationalarchives/jenkins-build-<name-of-node>:latest
   ```
 
-### Adding a new container
+#### Add a new container
 
 The docker container must start with `FROM jenkins/jnlp-slave` This image is mostly stock ubuntu and from there, you need to install whatever it is you need for your build. Build the docker image and push to docker hub.
 
  You then need to configure another container in the clouds section of the jenkins [configuration](docker/jenkins.yml) You can copy and paste most of it, just change the name and the image.
 
  Rebuild and push the jenkins docker container and redeploy to ECS. You can then use this container in your builds.
+
+## Backups
+
+Jenkins backups are run every weekday to save the Jenkins job configuration.
+
+We use [AWS Systems Manager Maintenance Windows][SMMW] to schedule and configure
+the backups. The Jenkins Terraform scripts in this repo configure a Bash script
+which is run on the Jenkins EC2 instance on a daily schedule.
+
+To investigate a failed or missed backup, see the [Maintenance Window history]
+in the management account.
+
+To manually initiate a backup, go the [Systems Manager Command History], select
```

Comment:
Assume it does not matter which previous backup command to run, provided it was a successful one?

Reply:
I'm not sure if it has to be successful, but it's probably easier to debug if you know you've rerun a successful backup.

This has made me think of something else - you wouldn't want to rerun a previous command if it was pointing to an old EC2 instance, or if you want to test some updated backup configuration.

I've updated the instructions a bit. Does it make sense?